justfile

```diff
@@ -16,6 +16,9 @@ test:
 test-e2e:
```

Comment:
is there still a need for `just test-e2e` with `test-e2e-standalone` present, or do they perform separate tasks?

I'm wondering if we can replace the github workflow using `just test-e2e` with `just test-e2e-standalone`.

Reply:
That's a good question @bacongobbler. I would keep `test-e2e` around for two reasons:

1. For now, the standalone tester is, er, untested. I would be gobsmacked if we didn't find issues, annoyances or limitations with it.  So, at least for a shakeout period, we need a reliable manual process to fall back on.

2. The standalone tester does not give the live feedback that the manual process does.  You don't see the kubelet logs whizzing by in their terminals; you don't see the tests passing and failing as they go.  This may be particularly relevant to "printf debugging" scenarios where the developer is adding ad hoc logging to the kubelet to figure out why something isn't working.

There is definitely a case for having `test-e2e` run the standalone tester, especially once we have ironed out the initial problems.  But I'd still want to keep, say, a `test-e2e-tests-only` target for those lower-level moments.